pgml-dashboard/src/api/docs.rs

```diff
@@ -24,54 +24,36 @@ async fn search(query: &str, index: &State<markdown::SearchIndex>) -> ResponseOk
-async fn doc_handler<'a>(path: PathBuf, cluster: &Cluster) -> Result<ResponseOk, Status> {
-    let guides = vec![
-        NavLink::new("Setup").children(vec![
-            NavLink::new("Installation").children(vec![
-                NavLink::new("v2").href("/docs/guides/setup/v2/installation"),
-                NavLink::new("Upgrade from v1.0 to v2.0")
-                    .href("/docs/guides/setup/v2/upgrade-from-v1"),
-                NavLink::new("v1").href("/docs/guides/setup/installation"),
-            ]),
-            NavLink::new("Quick Start with Docker")
-                .href("/docs/guides/setup/quick_start_with_docker"),
-            NavLink::new("Distributed Training").href("/docs/guides/setup/distributed_training"),
-            NavLink::new("GPU Support").href("/docs/guides/setup/gpu_support"),
-            NavLink::new("Developer Setup").href("/docs/guides/setup/developers"),
-        ]),
-        NavLink::new("Training").children(vec![
-            NavLink::new("Overview").href("/docs/guides/training/overview"),
-            NavLink::new("Algorithm Selection").href("/docs/guides/training/algorithm_selection"),
-            NavLink::new("Hyperparameter Search")
-                .href("/docs/guides/training/hyperparameter_search"),
-            NavLink::new("Preprocessing Data").href("/docs/guides/training/preprocessing"),
-            NavLink::new("Joint Optimization").href("/docs/guides/training/joint_optimization"),
-        ]),
-        NavLink::new("Predictions").children(vec![
-            NavLink::new("Overview").href("/docs/guides/predictions/overview"),
-            NavLink::new("Deployments").href("/docs/guides/predictions/deployments"),
-            NavLink::new("Batch Predictions").href("/docs/guides/predictions/batch"),
-        ]),
-        NavLink::new("Transformers").children(vec![
-            NavLink::new("Setup").href("/docs/guides/transformers/setup"),
-            NavLink::new("Pre-trained Models").href("/docs/guides/transformers/pre_trained_models"),
-            NavLink::new("Fine Tuning").href("/docs/guides/transformers/fine_tuning"),
-            NavLink::new("Embeddings").href("/docs/guides/transformers/embeddings"),
-        ]),
-        NavLink::new("Vector Operations").children(vec![
-            NavLink::new("Overview").href("/docs/guides/vector_operations/overview")
-        ]),
-        NavLink::new("Dashboard").href("/docs/guides/dashboard/overview"),
-        NavLink::new("Schema").children(vec![
-            NavLink::new("Models").href("/docs/guides/schema/models"),
-            NavLink::new("Snapshots").href("/docs/guides/schema/snapshots"),
-            NavLink::new("Projects").href("/docs/guides/schema/projects"),
-            NavLink::new("Deployments").href("/docs/guides/schema/deployments"),
-        ]),
-    ];
-
-    render(cluster, &path, guides, "Guides", &Path::new("docs")).await
+use rocket::fs::NamedFile;
+
+#[get("/docs/guides/.gitbook/assets/<path>", rank = 10)]
```

Comment:
You can use https://api.rocket.rs/v0.5-rc/rocket/fs/struct.FileServer.html

Reply:
Can't use the fileserver because we need the path to not be exact, but rather the relative gitbook path.